spanner/google/cloud/spanner/snapshot.py

```diff
@@ -157,9 +165,17 @@ class Snapshot(_SnapshotBase):
     :type exact_staleness: :class:`datetime.timedelta`
     :param exact_staleness: Execute all reads at a timestamp that is
                             ``exact_staleness`` old.
+
+    :type multi_use: :class:`bool`
+    :param multi_use: If true, the first read operation creates a read-only
```

Comment:
This is an implementation detail that its the first read which creates the transaction and need not be mentioned in the doc. You can just say
```
If true, multiple read/queries can be done with this Snapshot. They are all guaranteed to execute at the same timestamp.
```